app/web/routers/pipelines.py

```
@@ -58,19 +62,27 @@ def run_exercise_chat_pipeline_worker(dto: ExerciseChatPipelineExecutionDTO):
     status_code=status.HTTP_202_ACCEPTED,
     dependencies=[Depends(TokenValidator())],
 )
-def run_exercise_chat_pipeline(variant: str, dto: ExerciseChatPipelineExecutionDTO):
-    thread = Thread(target=run_exercise_chat_pipeline_worker, args=(dto,))
+def run_exercise_chat_pipeline(
+    variant: str,
+    event: str | None = Query(None, description="Event query parameter"),
+    dto: ExerciseChatPipelineExecutionDTO = Body(
+        description="Exercise Chat Pipeline Execution DTO"
+    ),
```

Ruff

Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)